org.adempiere.ui.zk/WEB-INF/src/web/theme/default/css/fragment/gadget.css.dsp

```diff
@@ -212,6 +212,10 @@
 	height: 100% !important; 
 }
 
+.statusline-gadget {
+	padding: 0px 10px 10px;
+}
+
 .help-content
```

Comment:
please update the "idempiere.theme.default" version in org.adempiere.ui.zk/WEB-INF/src/metainfo/zk/lang-addon.xml

Reply:
Hi @hengsin, fixed